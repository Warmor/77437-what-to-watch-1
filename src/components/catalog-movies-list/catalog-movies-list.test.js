import React from 'react';
import renderer from 'react-test-renderer';
import CatalogMoviesList from "~/components/catalog-movies-list/catalog-movies-list";

describe(`<CatalogMoviesList/>`, () => {
  it(`CatalogMoviesList renders correctly`, () => {
    const tree = renderer
      .create(<CatalogMoviesList filmsList={[]} onArticleTitleClick={jest.fn()}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

