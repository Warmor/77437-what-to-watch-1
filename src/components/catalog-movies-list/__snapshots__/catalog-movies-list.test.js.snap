// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CatalogMoviesList/> CatalogMoviesList renders correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <SmallMovieCard
    film={
      Object {
        "genreId": 1,
        "id": 1,
        "image": "img/fantastic-beasts-the-crimes-of-grindelwald.jpg",
        "title": "Fantastic Beasts",
        "video": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
      }
    }
    isActive={false}
    key="1"
    onArticleTitleClick={[MockFunction]}
    onMouseEnterCard={[Function]}
    onMouseLeaveCard={[Function]}
  />
</div>
`;
